class-tgm-plugin-activation.php

```diff
@@ -3108,6 +3108,7 @@ public function run( $options ) {
 					 *
 					 * @see Plugin_Upgrader::bulk_upgrade()
 					 * @see https://core.trac.wordpress.org/browser/tags/4.2.1/src/wp-admin/includes/class-wp-upgrader.php#L838
+					 * Last synced: Dec 31st 2015 against https://core.trac.wordpress.org/browser/trunk?rev=36134
```

Comment:
phpDocumentor, naturally (same developer), already follows the same format: http://www.phpdoc.org/docs/latest/references/phpdoc/tags/internal.html. 

https://github.com/WordPress/WordPress/blob/master/wp-includes/shortcodes.php#L17-L20 is one instance of the format in WP core, though they are other formats too that have not have been fixed up yet.